app/src/main/res/values/strings.xml

```diff
@@ -152,7 +152,6 @@
     <string name="ascend_edit_date">Datum:</string>
     <string name="ascend_edit_style">Stil:</string>
     <string name="ascend_edit_title">Begehung bearbeiten</string>
-    <string name="select_year">Jahr auswählen</string>
```

Comment:
Don't we display this anymore?

Reply:
Inflating the year picker layout into the dialog somehow overrides the complete dialog, including the title. Look good for me.